BrowserKit/Tests/TabDataStoreTests/TempTests.swift

```diff
@@ -4,8 +4,157 @@
 
 import XCTest
 @testable import TabDataStore
+import Common
 
 final class TempTests: XCTestCase {
```

Comment:
Would you mind renaming this to TabDataStore tests, this temp file was just added as a place holder to stop the compiler from complaining about the empty target.